### DISCUSSION TITLE
some zathura configurations

### DISCUSSION THREAD

**Discussion by payrim** [ahrm/sioyek#276] [Q&A]
(EDIT: first one is solved by startup commands) ~can i do default_mouse_drag_mode 1 ? is there another command for making mouse drag default when opening the program?~

the other issue i have is when i apply custom color.
![zathura](https://user-images.githubusercontent.com/62495972/173753322-0703b011-39e9-466b-80bd-acb20d49df00.png)
Sioyek's custom_text_color makes everything to the specified color which will almost remove all color in the text, rather than changing the blacks into the specified color only

is there a solution for this?  i believe dark mode does not kill the colors and change the background to black and the dark text to white only

(on sidenote, can i also change the color of that bar down there and menus? sorry i don't know what to search for those)

**Comment by ahrm** [comment]
We have a `toggle_dark_mode` command that tries to keep the color of document intact. In custom color mode, we do try to reproduce the colors using a combination of background and text color (this is by design, I don't consider this a bug). But for example if you set the background to black and foreground to white, there is no combination of these two colors that can reproduce a red hue. For example if you configure it like this you will see some colors:

```
custom_background_color		0.18 0.20 0.25
custom_text_color			1.0 1.0 1.0
```

> (on sidenote, can i also change the color of that bar down there and menus? sorry i don't know what to search for those)

You can configure it using `status_bar_color`.
